Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -2394,6 +2394,2519 @@ script:
+      predefined:
+      - Private
+      - Public
+      - All
+      required: false
+    - auto: PREDEFINED
+      description: Select the SDN connector you have created (dependencies - use fortiwebvm-sdn-connector-list command to get all the SDN connectors). Required when the server pool group server balance=Server Balance and server_type=External connector. Supports by API V2 only.
+      isArray: false
+      name: sdn_connector
+      predefined:
+      - aws
+      - azure
+      required: false
+    - description: Once you select the SDN collector that you have created, choose filter for your VMs in your public cloud account. You should know the filter name (there is no helper command). For example (AWS filter) instance-id=i-12345678. Required when the server pool group server balance=Server Balance and server_type=External connector. Supports by API V2 only.
+      isArray: false
+      name: filter
+      required: false
+    - description: 'The IP address of the web server to include in the pool. Required when: (server_type= ip)'
+      isArray: false
+      name: ip
+      required: false
+    - description: 'The fully-qualified domain name of the web server to include in the pool, such as www.example.com. Required when: (server_type=domain)'
+      isArray: false
+      name: domain
+      required: false
+    - description: "The TCP port number where the pool member listens for connections. The valid range is 1–65,535. "
+      isArray: false
+      name: port
+      required: false
+    - description: 'Connection limit, the maximum number of concurrent connections to the backend server. 0 for no connection limit.  Size range: 0-1048576'
+      isArray: false
+      name: connection_limit
+      required: false
+    - auto: PREDEFINED
+      description: Enable to allow HTTP/2 communication between the FortiWeb and this back-end web server for HTTP/2 security inspections.
+      isArray: false
+      name: http2
+      predefined:
+      - enable
+      - disable
+      required: false
+    - auto: PREDEFINED
+      description: Enable to use SSL/TLS for connections between FortiWeb and the pool member.
+      isArray: false
+      name: enable_ssl
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: 'The client certificate that FortiWeb uses to connect to this server pool member (dependencies - use fortiwebvm-local-certificate-list command to get all the local certificates). Relevant when: ssl=enable.'
+      isArray: false
+      name: client_certificate_file
+      required: false
+    - description: 'Recover, seconds to postpone forwarding traffic after downtime when a health check indicates that this server has become available again. Size range: 0-86400'
+      isArray: false
+      name: recover
+      required: false
+    - description: 'Warm up, if the server cannot initially handle full connection load when it begins to respond to health checks. Size range: 0-86400'
```

Comment:
Not sure what this description should be. Perhaps:

'The number of seconds to wait if the server cannot initially handle full connection load when it begins to respond to health checks. Size range: 0-86400.'